frontend/packages/client/package.json

```diff
@@ -4,15 +4,16 @@
   "private": true,
   "dependencies": {
     "@creativebulma/bulma-tooltip": "^1.2.0",
-    "@onflow/fcl": "^0.0.77",
     "@onflow/config": "0.0.2",
+    "@onflow/fcl": "^0.0.77",
     "@onflow/six-transfer-tokens": "0.0.8",
     "bulma": "0.9.3",
     "cookies-js": "^1.2.3",
     "draft-js": "^0.11.7",
     "draftjs-to-html": "^0.9.1",
     "lodash": "4.17.21",
     "millify": "^4.0.0",
+    "nanoid": "^3.3.4",
```

Comment:
using this package to create a unique slug required by the backend to create community